content-api/hierarchy-manager/src/main/scala/org/sunbird/managers/HierarchyManager.scala

```diff
@@ -608,9 +608,9 @@ object HierarchyManager {
 
     def validateLeafNodes(parentNode: java.util.Map[String, AnyRef], childNode: java.util.Map[String, AnyRef])(implicit oec: OntologyEngineContext, ec: ExecutionContext) = {
         val primaryCategory = parentNode.getOrDefault("primaryCategory", "").asInstanceOf[String]
-        val channel = parentNode.getOrDefault("channel", "_all")
-        val categoryId = if (StringUtils.isBlank(primaryCategory)) "" else "obj-cat:" + Slug.makeSlug(primaryCategory + "_" + parentNode.getOrDefault("objectType", "").asInstanceOf[String].toLowerCase() + "_" + channel)
-        val outRelations = DefinitionNode.getOutRelations(HierarchyConstants.GRAPH_ID, "1.0", parentNode.getOrDefault("objectType", "").asInstanceOf[String].toLowerCase().replace("image", ""), categoryId)
+        val channel: String = parentNode.getOrDefault("channel", "_all").asInstanceOf[String]
```

Comment:
```suggestion
        val channel: String = parentNode.getOrDefault("channel", "all").asInstanceOf[String]
```

I think the default value should be `all` not `_all`.